### DISCUSSION TITLE
Self-signed certificate

### DISCUSSION THREAD

**Discussion by fung** [apideck-libraries/portman#454] [Q&A] [answered]
Hi,

May I know how to ignore SSL / TLS certificate issue?

The output I got:
`unable to get local issuer certificate`

Thanks!

**Comment by thim81** [comment, marked as answer]
hi @fung 

If you are using Postman, you can set the setting "SSL certificate verification" OFF

![image](https://user-images.githubusercontent.com/952446/216769288-1cd30bd7-16fe-4542-9694-7a711d83f060.png)

If you are using Newman CLI to execute requests, have a look at our example: 
https://github.com/apideck-libraries/portman/blob/main/examples/cli-options/readme.md#portman-newman-cli-options

The setting: `"insecure": true` will turn on of the SSL certificate validation.

**Reply by fung** [reply]
Thanks @thim81 !
